packages/ui-react/src/pages/User/User.tsx

```diff
@@ -87,29 +72,17 @@ const User = (): JSX.Element => {
         <nav className={styles.leftColumn}>
           <div className={styles.panel}>
             <ul>
-              {accessModel === 'SVOD' && profilesEnabled && profileAndFavoritesPage && (
```

Comment:
Should we keep `accessModel` here?

Reply:
Looks like it can be removed

Reply:
It is removed, you're looking at a deleted line. 😃

Reply:
Yeah, that is what I meant. I think we are good here

Reply:
Ah, got it. Yeah, this entire block is removed, not just the condition we are looking at.